src/io/copy.php

```diff
@@ -0,0 +1,36 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\IO;
+
+use namespace HH\Lib\Str;
+
+/**
+ * Copies content from the given $source read handle to the $target write handle
+ * until the eol of $source is reached.
+ *
+ * If `$chunk_size` is `null`, there is no limit on the chunk size - this function will
+ * copy the content of $source all at once.
+ */
+async function copy(
```

Comment:
To match the naming conventions, `copyAsync()` or is this handled by the `gen` rewriter?

Reply:
not sure about this. other functions in `File\` don't use the `_async` suffix.

Reply:
it should be copy_async

Other functions in File\ do not return Awaitables, so do not have the suffix (if that's not the case, it's a bug that needs fixing)